upup/pkg/fi/files_owner.go

```diff
@@ -0,0 +1,58 @@
+// +build !windows
```

Comment:
Don't build this if we are compiling for windows.  This code is only used by `nodeup`.